src/dbc_grammar.peg

```diff
@@ -13,16 +13,16 @@ message                 <- 'BO_' s* number s* TOKEN ':' s number s TOKEN _ signa
 bo_tx_bu                <- < 'BO_TX_BU_' s* number s* ':' s* ECU_TOKEN (',' ECU_TOKEN)* ';' > NewLine
 cm                      <- < 'CM_' s* phrase > ';' NewLine
 cm_bu                   <- < 'CM_' s* 'BU_' s* TOKEN s* phrase > ';' NewLine
-cm_bo                   <- < 'CM_' s* 'BO_' s* integer s* phrase > ';' NewLine
-cm_sg                   <- < 'CM_' s* 'SG_' s* integer s* TOKEN s* phrase > ';' NewLine*
+cm_bo                   <- < 'CM_' s* 'BO_' s* number s* phrase > ';' NewLine
```

Comment:
Yes, you're right. But this would require rewriting quite a bit more of this parser, which is why I just stuck with number -- see also the commit message in https://github.com/Proemion/CANdb/pull/1/commits/b3d97938620e82a63936e3241ff5bbf3c7ee98d4